frontend/css/layout.css

```diff
@@ -0,0 +1,110 @@
+
+.container {
+   display: flex;
+}
+
+.grid {
+   display: grid;
+}
+/* Account management */
+
+.account {
+   justify-content: flex-end;
+   padding-right: 20px;
+}
+
+.account-links {
+  display: flex;
+  list-style: none;
+  gap: 20px;
+}
+
+/* Search section */
+
+.search {
+   padding: 20px;
+}
+
+.logo {
+   flex:1;
+   display: flex;
+   justify-content: flex-start;
+   align-items: center;
+}
+
+.search-bar{
+   flex: 1;
+   display: flex;
+   justify-content: center;
+   align-items: center;
+}
+
+#searchbar {
```

Comment:
Just a comment on selectors in general, I prefer to not target elements by id just to avoid specificity conflicts and help with reusability. 

In this case perhaps we can also set the width to be a percentage of its container, or let the container specify that relative width? In order to help with keeping the content flexible depending on the viewport width.